libs/application-generic/src/logging/index.ts

```diff
@@ -107,7 +105,7 @@ export function createNestLoggingModuleOptions(
   // eslint-disable-next-line no-console
   console.log(
     `Selected Log Transport ${!transport ? 'None' : 'pino-pretty'}`,
-    loggingLevelSet,
+    loggingLevelSet
```

Comment:
It looks like your Eslint configuration isn't working correctly. Can you update your IDE settings to disable the builtin IDE formatter and try again? I'm expecting to see a comma present at the end of this line (and lines above that have formatting changes)